app/component/StopPageActionBar.js

```diff
@@ -19,7 +24,17 @@ const StopPageActionBar = ({
         dateFormat={DATE_FORMAT}
         onDateChange={onDateChange}
       />
-      <PrintLink className="action-bar" href={printUrl} />
+      <div className="print-button-container">
+        <SecondaryButton
+          buttonParams={{
```

Comment:
Is there a reason why all params are wrapped in an object?

Reply:
No particular reason, just a preference of having a generic component getting the props as a "package" instead of multiple pieces for maintenance reasons.